pkg/graveler/graveler.go

```diff
@@ -51,7 +51,8 @@ type RawRef struct {
 }
 
 type DiffSummary struct {
-	Count map[DiffType]int
+	Count       map[DiffType]int
+	MissingInfo bool // true when Diff summary has missing Information (could happen when skipping ranges with same bounds)
```

Comment:
Incomplete
```suggestion
	Incomplete bool // true when Diff summary has missing Information (could happen when skipping ranges with same bounds)
```

Should this also be used on the command line when showing the merge summary?

Reply:
Changed.
Not as part of this PR.